internal/kafka/test/integration/require_terms_acceptance_middleware_test.go

```diff
@@ -123,4 +124,22 @@ func TestTermsRequired_ListKafkaTermsRequired(t *testing.T) {
 
 	Expect(err).NotTo(HaveOccurred())
 	Expect(resp.StatusCode).To(Equal(http.StatusOK))
+
+	clusterID, getClusterErr := common.GetRunningOsdClusterID(env.helper, t)
+	if getClusterErr != nil {
+		t.Fatalf("Failed to retrieve cluster details: %v", getClusterErr)
+	}
+	if clusterID == "" {
+		panic("No cluster found")
```

Comment:
I see lot of `panic` in this code. Is this correct?
Wouldn't be better to use
```go
Expect(clusterID).ToNot(Equal(""))
```
?